### PR TITLE
[KOGITO-3510] Deploy/Promote pipelines: set credentials as params

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3510

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
Tested with https://github.com/kiegroup/kogito-pipelines/pull/68

File: Jenkinsfile.promote [hunk 1]
```diff
@@ -40,15 +40,15 @@ pipeline {
         booleanParam(name: 'BASE_IMAGE_USE_OPENSHIFT_REGISTRY', defaultValue: false, description: 'Override `deployment.properties`. Set to true if base image should be retrieved from Openshift registry.In this case, BASE_IMAGE_REGISTRY_CREDENTIALS, BASE_IMAGE_REGISTRY and BASE_IMAGE_NAMESPACE parameters will be ignored')
         string(name: 'BASE_IMAGE_REGISTRY_CREDENTIALS', defaultValue: '', description: 'Override `deployment.properties`. Base Image registry credentials to use to deploy images. Will be ignored if no BASE_IMAGE_REGISTRY is given')
         string(name: 'BASE_IMAGE_REGISTRY', defaultValue: '', description: 'Override `deployment.properties`. Base image registry')
-        string(name: 'BASE_IMAGE_NAMESPACE', defaultValue: 'kiegroup', description: 'Override `deployment.properties`. Base image namespace')
```
Comment (sutaakar) [inline Jenkinsfile.promote:43]: Why the `kiegroup` is removed from default value?

Comment (radtriste) [reply Jenkinsfile.promote:43]: as description mention, it overrides value taken from `deployment.properties` of deploy job if defined. So we should not have a default value, else this default value will always override and we would not take the right namespace for the image

Comment (sutaakar) [reply Jenkinsfile.promote:43]: ok, thanks

**Review by sutaakar — APPROVED** [review]

**Review by Ginxo — APPROVED** [review]
LGTM

**Review by spolti — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

**Review by Kevin-Mok — APPROVED** [review]

**Pull request merged**